packages/kit/src/core/build/build_client.js

```diff
@@ -89,6 +89,11 @@ export async function build_client({
 		plugins: [
 			svelte({
 				extensions: config.extensions,
+				// In AMP mode, we know that there are no conditional component imports. In that case, we
+				// don't need to include CSS for components that are imported but unused, so we can just
+				// include rendered CSS.
+				// This would also apply if hydrate and router are both false, but we don't know if one
+				// has been enabled at the page level, so we don't do anything there.
```

Comment:
would it be worth deduping these?

Reply:
I might except I figure we're going to remove `config.kit.amp` which means a lot of this will get deleted, so I don't know if it's really worth it